web/custom-plugins/forecast-of-response/src/Pages/OperationCreate.tsx

```diff
@@ -128,6 +100,7 @@ const getEntityId = (
 const createOperationEntity = (
   operationName: string,
   operationLabel: string,
+  SIMACompute: File,
```

Comment:
seems like the multiline comment above is missing description of  operationLabel and SIMACompute. 
We should probably update the comment or remove it if ts difficult to maintain.

Reply:
updated!